src/app-layout/__tests__/trigger-button.test.tsx

```diff
@@ -439,27 +411,18 @@ describe('Visual Refresh Toolbar trigger-button', () => {
             isMobile,
           });
           expect(getByTestId(mockTestId)).toBeTruthy();
-          expect(
-            wrapper!.getElement().classList.contains(toolbarTriggerButtonStyles['trigger-wrapper-tooltip-visible'])
-          ).toBe(false);
-          expect(wrapper!.findByClassName(toolbarTriggerButtonStyles['trigger-tooltip'])).toBeNull();
+          expect(wrapper!.findByClassName(testUtilStyles['trigger-tooltip'])).toBeNull();
           expect(() => getByText(mockTooltipText)).toThrow();
           fireEvent.focus(wrapper!.getElement());
           expect(getByText(mockTooltipText)).toBeTruthy();
-          expect(
-            wrapper!.getElement().classList.contains(toolbarTriggerButtonStyles['trigger-wrapper-tooltip-visible'])
-          ).toBe(true);
 
           fireEvent.keyDown(wrapper!.getElement(), {
             ...mockEventBubble,
             key: 'Escape',
             code: KeyCode.escape,
           });
 
-          expect(wrapper.findByClassName(toolbarTriggerButtonStyles['trigger-tooltip'])).toBeNull();
-          expect(
-            wrapper!.getElement().classList.contains(toolbarTriggerButtonStyles['trigger-wrapper-tooltip-visible'])
-          ).toBe(false);
+          expect(wrapper.findByClassName(testUtilStyles['trigger-tooltip'])).toBeNull();
```

Comment:
Let's create a variable for `testUtilStyles['trigger-tooltip']` to eliminate this repetitive code